app/models/user.rb

```diff
@@ -0,0 +1,10 @@
+class User < ApplicationRecord
+    before_save { self.email = email.downcase }
+    validates :name,  presence: true, length: { maximum: 50 }
+    VALID_EMAIL_REGEX = /\A[\w+\-.]+@[a-z\d\-.]+\.[a-z]+\z/i
+    validates :email, presence: true, length: { maximum: 255 }, 
+        format: { with: VALID_EMAIL_REGEX },
```

Comment:
```suggestion
      format: { with: VALID_EMAIL_REGEX },
      uniqueness: true
```
nếu code xuống dòng cách 2 space nhé.